pycbc/inference/distributions.py

```diff
@@ -568,53 +699,41 @@ def rvs(self, size=1, param=None):
 
 
 class CosAngle(SinAngle):
-    """
-    A sine distribution between the given bounds, which must be within
-    [-pi/2,pi/2). If no bounds are provided, will default to [-pi/2, pi/2). The
-    parameter bounds are initialized as multiples of pi, while the stored
-    bounds are in radians.
-
-    The pdf of each parameter `\theta` is given by:
-    ..math::
-        p(\theta) = \frac{\cos \theta}{\sin\theta_0 - \sin\theta_1}, \theta_0 \leq \theta < \theta_1,
-
-    and 0 otherwise. Here, :math:`\theta_0, \theta_1` are the bounds of the
-    parameter.
+    r"""A cosine distribution. This is the same thing as a sine distribution,
+    but with the domain shifted to `[-pi/2, pi/2]`. See SinAngle for more
+    details.
 
     Parameters
     ----------
     \**params :
         The keyword arguments should provide the names of parameters and
-        (optionally) their corresponding bounds, as tuples. The bounds must be
-        in [-0.5,0.5). These are converted to radians for storage.
-        None may also be passed; in that case, the parameter will sine
-        distributed between -pi/2 and pi/2.
+        (optionally) their corresponding bounds, as either
+        `boundary_utils.Bounds` instances or tuples. The bounds must be
+        in [-0.5, 0.5]. These are converted to radians for storage.
+        None may also be passed; in that case, the domain bounds will be used.
 
     Class Attributes
     ----------------
-    name : 'cos_angle'
+    name : 'sin_angle'
```

Comment:
I think you accidently copied ``name : 'sin_angle'`` here. Can you change it back to ``name : 'cos_angle'``?